README.md

```diff
@@ -19,6 +19,8 @@ Once everything is ready, `kubectl get service jaeger-query` tells you where to
 If you are using `minikube` to setup your Kubernetes cluster, the command `minikube service jaeger-query --url`
 can be used instead.
 
+As the Jaeger Agent is deployed with the other components, your application needs to tell the Jaeger Client where to find the agent (`jaeger-all-in-one-agent`). Refer to your client's documentation for the appropriate mechanism, but most clients allow this to be set via the environment variable `JAEGER_AGENT_HOST`.
```

Comment:
I would like to see this in a separate section. The guide should tell what service names should be used for HTTP and UDP sender.

is there any service named like `jaeger-all-in-one-agent`?

Reply:
In the k8s resource files in this repository https://github.com/jaegertracing/jaeger-kubernetes/blob/master/jaeger-production-template.yml#L65. 

The agent might be more complicated as it depends how the agent is deployed

Reply:
Yes, it looks good. Only `jaeger-collector` and `zipkin` are for collecting. `jaeger-query` is for querying data so not important here.

The jaeger-collector port is `14268`, zipkin port is `9411`.